ansible/pbTestScripts/buildJDKWin.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+export TARGET_OS=linux
```

Comment:
Is linux right?

Reply:
I tried it with `windows` but it just gave error messages, seems to work with `linux` though

Reply:
That doesn't sound right at all - what error messages?

Reply:
Actually! My mistake- Just retried it and it seems to work fine.